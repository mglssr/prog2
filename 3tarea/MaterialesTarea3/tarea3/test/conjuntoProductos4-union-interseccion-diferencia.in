crearConjuntoProductos 10
insertarConjuntoProductos 0
insertarConjuntoProductos 1
insertarConjuntoProductos 2
insertarConjuntoProductos 3
insertarConjuntoProductos 6
insertarConjuntoProductos 7
insertarConjuntoProductos 8
insertarConjuntoProductos 9
imprimirConjuntoProductos
unionConjuntoProductos 5 1 3 4 5 6
interseccionConjuntoProductos 5 1 3 4 5 6
diferenciaConjuntoProductos 5 1 3 4 5 6
liberarConjuntoProductos
Fin
