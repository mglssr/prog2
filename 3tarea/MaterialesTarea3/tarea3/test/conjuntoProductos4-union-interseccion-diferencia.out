Bienvenido al programa principal de MercadoFinger. Por favor ingrese su comando:
1> El conjunto productos fue creado en forma correcta.
2> Se ejecuto 'insertarTConjuntoProductos' con producto de id 0 exitosamente.
3> Se ejecuto 'insertarTConjuntoProductos' con producto de id 1 exitosamente.
4> Se ejecuto 'insertarTConjuntoProductos' con producto de id 2 exitosamente.
5> Se ejecuto 'insertarTConjuntoProductos' con producto de id 3 exitosamente.
6> Se ejecuto 'insertarTConjuntoProductos' con producto de id 6 exitosamente.
7> Se ejecuto 'insertarTConjuntoProductos' con producto de id 7 exitosamente.
8> Se ejecuto 'insertarTConjuntoProductos' con producto de id 8 exitosamente.
9> Se ejecuto 'insertarTConjuntoProductos' con producto de id 9 exitosamente.
10> Conjunto de productos:
0 1 2 3 6 7 8 9 
11> Conjunto de productos union:
0 1 2 3 4 5 6 7 8 9 
12> Conjunto de productos interseccion:
1 3 6 
13> Conjunto de productos diferencia:
0 2 7 8 9 
14> Conjunto de productos liberado exitosamente.
15> Fin.
